framework/metric.go

```diff
@@ -5,17 +5,18 @@ import (
 )
 
 var (
-	frameworkHistogram = prometheus.NewHistogramVec(
-		prometheus.HistogramOpts{
-			Namespace: "operatorkit",
-			Subsystem: "framework",
-			Name:      "event",
-			Help:      "Histogram for events within the operatorkit framework.",
+	frameworkEventSummary = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
+			Namespace:  "operatorkit",
+			Subsystem:  "framework",
+			Name:       "event",
+			Help:       "Summary for events within the operatorkit framework.",
+			Objectives: map[float64]float64{},
```

Comment:
Judging from godoc, I would assume that setting `Objectives` to the empty map, you don't measure any quantiles?

Reply:
Feels like it now. When I read the docs I thought setting an empty map will measure it automatically based on the times being observed. When we need to define buckets again we can just drop this because specific buckets are the reason this metric does not seem to work here. I actually just want to observe timestamps for the operator events since the framework events are executed in sequence and I realized percentiles do not make sense because we only have one event at a time anyway. So calculating the percentiles for something that is unique on a timeline is pointless.

Reply:
This makes sense to me.